boto/vendored/regions/exceptions.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Chatted offline about this.  Given how infrequently we've updated this class in botocore, I'm good with leaving the structure as is.